### PR TITLE
fix plot source typo in label

### PR DESCRIPTION
**Pull request by mloubout**
TIny typo fix

### CODE REVIEW

File: examples/seismic/source.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def show(self, idx=0, time=None, wavelet=None):
         plt.figure()
         plt.plot(time, wavelet)
         plt.xlabel('Time (ms)')
-        plt.ylabel('Velocity (km/s)')
+        plt.ylabel('Amplitude')
```
Comment (mlange05) [inline examples/seismic/source.py]: No unit?

Comment (mloubout) [reply examples/seismic/source.py]: Would depend on the kernel, acoustic would be pressure, but elastic wouldn't. Better left without any in my opinion.

Comment (mlange05) [reply examples/seismic/source.py]: ok, fair enough. Good to go then.

File: examples/seismic/tti/operators.py [hunk 2]
```diff
@@ -100,7 +100,7 @@ def Gyy_shifted(field, cosphi, sinphi, space_order):
     return -.5 * (Gyy + Gyy2)
 
 
-def Gzz_shited(field, costheta, sintheta, cosphi, sinphi, space_order):
+def Gzz_shifted(field, costheta, sintheta, cosphi, sinphi, space_order):
```
Comment (mlange05) [inline examples/seismic/tti/operators.py]: :joy: :joy: :joy:

Comment (mloubout) [reply examples/seismic/tti/operators.py]: Yep, nicely noticed byt tim 😅

**Comment by mloubout** [comment]
Only failing on doc

**Comment by FabioLuporini** [comment]
Merged.

**Pull request merged**